RELEASE_PROCESS.md

```diff
@@ -0,0 +1,27 @@
+# How to cut a peg release
+
+This document is aimed at members of the Pony team who might be cutting a release of Pony. It serves as a checklist that can take you through doing a release step-by-step.
+
+## Prerequisites
+
+You must have commit access to the peg repository
+
+## Releasing
+
+Please note that this document was written with the assumption that you are using a clone of the `peg` repo. You have to be using a clone rather than a fork. It is advised to your do this by making a fresh clone of the `peg` repo from which you will release.
```

Comment:
```suggestion
Please note that this document was written with the assumption that you are using a clone of the `peg` repo. You have to be using a clone rather than a fork. It is advised that you do this by making a fresh clone of the `peg` repo from which you will release.
```

Reply:
Welp, that mistake is in a ton of other documents.

Reply:
I've missed it too.